snomed/com.b2international.snowowl.snomed.importer.rf2/src/com/b2international/snowowl/snomed/importer/rf2/SnomedCompositeImporter.java

```diff
@@ -281,48 +268,39 @@ private boolean isRefSetImport(final Iterable<? extends ComponentImportUnit> uni
 	
 	@Override
 	public void postImport(final SubMonitor subMonitor) {
-	
 		subMonitor.setWorkRemaining(importers.size());
 		
 		for (final Importer importer : importers) {
 			importer.postImport(subMonitor.newChild(1, SubMonitor.SUPPRESS_NONE));
 		}
 		
-		if (getImportBranchPath().getPath().equals(IBranchPath.MAIN_BRANCH)) {
-			createNewVersions(importContext.getEditingContext());
-		} else {
-			try (SnomedEditingContext snomedEditingContext = new SnomedEditingContext(BranchPathUtils.createMainPath())) {
-				createNewVersions(snomedEditingContext);
-			}
-		}
+		createNewVersions();
 	}
 
-	private void createNewVersions(SnomedEditingContext snomedEditingContext) {
-		try {
-			final CodeSystem codeSystem = checkNotNull(snomedEditingContext.lookup(importContext.getCodeSystemShortName(), CodeSystem.class));
+	private void createNewVersions() {
+		
+		try (SnomedEditingContext codeSystemEditingContext = new SnomedEditingContext(BranchPathUtils.createMainPath())) {
```

Comment:
We've talked earlier this week about moving the code system versions to the code system`s `MAIN` branch. Should we implement this as part of this PR?